flood_adapt/object_model/interface/events.py

```diff
@@ -137,11 +137,14 @@ class SurgeSource(str, Enum):
     shape = "shape"
 
 
+class SurgeShape(Enum):
+    gaussian = "gaussian"
+
```

Comment:
There is a ShapeType enum for this in this file

Reply:
sorry missed that